src/api/subscriptions.js

```diff
@@ -115,6 +115,12 @@ export class Subscriptions {
    */
   subscribe(skuOrSubscriptionRequest) {}
 
+  /**
+   * Starts the Contributions flow.
```

Comment:
I'm not entirely clear we are ready to define top-level API here. Can we defer this until we have the flow a bit more flashed out? Alternatively, we should put it under the experiment. Otherwise, I'd rather not have the new unprotected API appearing in the PROD API.

Reply:
Removed for now from the API.